@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/cfgFunctions.hpp

```diff
@@ -64,8 +72,9 @@ class CfgFunctions
         };
         class simpleRotor {
             file = "\fza_ah64_sfmplus\functions\simpleRotor";
-            class simpleRotor {R;};
+            class simpleRotorMain {R;};
             class simpleRotorNewtRaphSolver {R;};
+            class simpleRotorTail {R;};
```

Comment:
Duplicated in Tailrotor Pr

Reply:
Removing them would break this PR

Reply:
I dont entierely understand how, as this in itself would create a missing file error, but sure